src/merkle/simple_smt/mod.rs

```diff
@@ -6,7 +6,7 @@ mod tests;
 // SPARSE MERKLE TREE
 // ================================================================================================
 
-/// A sparse Merkle tree with 63-bit keys and 4-element leaf values, without compaction.
+/// A sparse Merkle tree with 64-bit keys and 4-element leaf values, without compaction.
```

Comment:
Sorry I missed this - when did we switch from 63-bit to 64-bit keys. Do you happen to have a reference to the discussion?

Reply:
There isn't a discussion over that. The idea is to make it consistent with the RPO module documentation and arguments of the `SimpleSmt` (that will take a `u64`).

https://github.com/0xPolygonMiden/crypto/blob/398af59045efe16181140475742c7535618a9c75/src/hash/rpo/mod.rs#L60-L61

I'm reverting to the previous `63` so we can retake this topic for the next release.

Reply:
Added #65